src/core/dwn-error.ts

```diff
@@ -20,10 +20,6 @@ export enum DwnErrorCode {
   AuthorizationNotGrantedToAuthor = 'AuthorizationNotGrantedToAuthor',
   ComputeCidCodecNotSupported = 'ComputeCidCodecNotSupported',
   ComputeCidMultihashNotSupported = 'ComputeCidMultihashNotSupported',
-  DidMethodNotSupported = 'DidMethodNotSupported',
```

Comment:
Found a few unused error codes, so I went through and removed others.

Reply:
😮 🙏
Did you write a script for this?

Reply:
No script, lol Although it would be nice to validate this during CI/lint.

But I spotted a few i knew weren't there and decided to go through the list haha.